styles/InfoCard.module.css

```diff
@@ -0,0 +1,25 @@
+.ICImage {
+    max-width: 380px;
+    max-height: 264px;
+    border-radius: 6px;
+    object-fit: cover;
+}
+
+.ICBody {
+    max-width: 379px;
+    max-height: 350px;
+    font-size: 20px;
+    line-height: 35px;
+    letter-spacing: 0.05em;
```

Comment:
ooh some of this can definitely be pulled into the global styles.. out of scope of this PR, but something to look into!